apps/nextjs/tailwind.config.ts

```diff
@@ -3,6 +3,9 @@ import type { Config } from "tailwindcss";
 import baseConfig from "@acme/tailwind-config";
 
 export default {
-  content: baseConfig.content,
+  content: [
+    ...baseConfig.content,
+    "../../packages/ui/src/**/*.{js,jsx,ts,tsx,mdx}",
```

Comment:
cmon - we're ts only here

```suggestion
    "../../packages/ui/src/**/*.tsx",
```